media_softlet/agnostic/common/vp/hal/features/vp_hdr_filter.cpp

```diff
@@ -167,8 +167,8 @@ MOS_STATUS VpHdrFilter::CalculateEngineParams(
                 if (argHandle == m_renderHdr3DLutOclParams.end())
                 {
                     KRN_ARG krnArg = {};
-                    argHandle      = m_renderHdr3DLutOclParams.insert(std::make_pair(uIndex, krnArg)).first;
-                    VP_PUBLIC_CHK_NOT_FOUND_RETURN(argHandle, &m_renderHdr3DLutOclParams);
+                    argHandle      = m_renderHdr3DLutOclParams.emplace(uIndex, krnArg).first;
+                    VP_PUBLIC_CHK_NOT_FOUND_RETURN(argHandle, &m_renderHdr3DLutL0Params);
```

Comment:
There is compilation error caused by this line

Reply:
Fix it https://github.com/intel/media-driver/pull/1875/commits/c95e85f663c1796f102f223f5577bcbc2e38ee98